corehq/apps/export/models/new.py

```diff
@@ -30,20 +34,20 @@ class ExportItem(DocumentSchema):
     path = ListProperty()
     label = StringProperty()
     tag = StringProperty()
-    last_occurrence = IntegerProperty()
+    last_occurrences = DictProperty()
```

Comment:
Looks like the doc string for `ExportItem` could also use updating for a couple of its properties.